assets/control-plane/service-monitor-etcd.yaml

```diff
@@ -14,10 +14,10 @@ spec:
       caFile: /etc/prometheus/secrets/kube-etcd-client-certs/etcd-client-ca.crt
       certFile: /etc/prometheus/secrets/kube-etcd-client-certs/etcd-client.crt
       keyFile: /etc/prometheus/secrets/kube-etcd-client-certs/etcd-client.key
-  jobLabel: k8s-app
+  jobLabel: app.kubernetes.io/name
   namespaceSelector:
     matchNames:
     - openshift-etcd
   selector:
     matchLabels:
-      k8s-app: etcd
+      app.kubernetes.io/name: etcd
```

Comment:
this is causing the test failure. AFAICT the etcd service is managed by the etcd operator and the label selector shouldn't be changed.